source/FilesServer/app/endpoints.py

```diff
@@ -1,12 +1,16 @@
 from fastapi import APIRouter, UploadFile, status, HTTPException
 from fastapi.responses import RedirectResponse
 from app import service
+from app.models import FileMetadata
 
 router = APIRouter()
 
 
+# TODO - Check how to use Depends in order not to repeat get_settings() and get_client in each endpoint
 @router.get("/download", status_code=status.HTTP_200_OK)
-async def download(file_name: str) -> RedirectResponse:
+async def download(
```

Comment:
reformat the code

Reply:
i used ruff format . and ruff check . --fix and this is the outcome.. what am i doing wrong?

Reply:
ok, maybe i'm blind but this line looked a bit short to me. not long enough to be splitted